### PR TITLE
Dropout

### LINKED ISSUE #57: remove dropout option from CuDNNLSTM layers
The author of keras, Francois Chollet, suggests that using regular dropout in stacked RNNs will "wreck training" (https://github.com/keras-team/keras/issues/8935)

### PR DESCRIPTION
**Pull request by mr-francois**
resolve #57

### CODE REVIEW

**Comment by mr-francois** [comment]
resolve #57
